src/view/edit-point-view.js

```diff
@@ -1,157 +1,171 @@
-        </div>
-
-        <div class="event__field-group  event__field-group--price">
-          <label class="event__label" for="event-price-1">
-            <span class="visually-hidden">Price</span>
-            &euro;
-          </label>
-          <input class="event__input  event__input--price" id="event-price-1" type="text" name="event-price" value="160">
-        </div>
-
-        <button class="event__save-btn  btn  btn--blue" type="submit">Save</button>
-        <button class="event__reset-btn" type="reset">Delete</button>
-        <button class="event__rollup-btn" type="button">
-          <span class="visually-hidden">Open event</span>
-        </button>
-      </header>
-      <section class="event__details">
-        <section class="event__section  event__section--offers">
-          <h3 class="event__section-title  event__section-title--offers">Offers</h3>
-
-          <div class="event__available-offers">
-            <div class="event__offer-selector">
-              <input class="event__offer-checkbox  visually-hidden" id="event-offer-luggage-1" type="checkbox" name="event-offer-luggage" checked>
-              <label class="event__offer-label" for="event-offer-luggage-1">
-                <span class="event__offer-title">Add luggage</span>
-                &plus;&euro;&nbsp;
-                <span class="event__offer-price">50</span>
-              </label>
-            </div>
-
-            <div class="event__offer-selector">
-              <input class="event__offer-checkbox  visually-hidden" id="event-offer-comfort-1" type="checkbox" name="event-offer-comfort" checked>
-              <label class="event__offer-label" for="event-offer-comfort-1">
-                <span class="event__offer-title">Switch to comfort</span>
-                &plus;&euro;&nbsp;
-                <span class="event__offer-price">80</span>
-              </label>
+import dayjs from 'dayjs';
+import { getRandomInteger } from '../utils.js';
+
+let offersHeaderClass;
+
+const createChooseDestinationTemplate = (destination) =>
+  `<input class="event__input  event__input--destination" id="event-destination-1" type="text" name="event-destination" value="${destination.name}" list="destination-list-1">
+  <datalist id="destination-list-1">
+    <option value="Amsterdam"></option>
+    <option value="Geneva"></option>
+    <option value="Chamonix"></option>
+  </datalist>`;
+
+const createTypeIconTemplate = (type) =>
+  `<img class="event__type-icon" width="17" height="17" src="img/icons/${type}.png" alt="Event type icon">`;
+
+const createEditPointDateTemplate = (dateFrom, dateTo) =>
+  `<label class="visually-hidden" for="event-start-time-1">From</label>
+  <input class="event__input  event__input--time" id="event-start-time-1" type="text" name="event-start-time" value="${dayjs(
+    dateFrom
+  ).format('DD/MM/YYYY HH:mm')}">
```

Comment:
```suggestion
const createEditPointDateTemplate = (dateFrom, dateTo) => (
  `<label class="visually-hidden" for="event-start-time-1">From</label>
  <input 
    class="event__input  event__input--time" 
    id="event-start-time-1"
    type="text" 
    name="event-start-time" 
    value="${dayjs(dateFrom).format('DD/MM/YYYY HH:mm')}"
  >
```

${dayjs(dateFrom).format('DD/MM/YYYY HH:mm')} в переменную, ну и формат дат собери в перечисление)